src/components/utils/blocks.ts

```diff
@@ -23,6 +26,22 @@ export function isBlockConvertable(block: Block, direction: 'export' | 'import')
 /**
  * Checks that all the properties of the first block data exist in second block data with the same values.
  *
+ * If we have several entries with their own data overrides,
+ * find those who matches some current data property
+ *
+ * Example:
+ *  Tools' toolbox: [
+ *    {title: "Heading 1", data: {level: 1} },
+ *    {title: "Heading 2", data: {level: 2} }
+ *  ]
+ *
+ *  the Block data: {
+ *    text: "Heading text",
+ *    level: 2
+ *  }
+ *
+ *  that means that for the current block, the second toolbox item (matched by "{level: 2}") is active
+ *
```

Comment:
Are you sure this comment should be there? Seems like it is unrelated to the following function

Reply:
updated comment